CHANGELOG.md

```diff
@@ -120,6 +120,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
+- Add `AUPIMO` tutorials notebooks in https://github.com/openvinotoolkit/anomalib/pull/2330 and https://github.com/openvinotoolkit/anomalib/pull/2336
```

Comment:
@jpcbertoldo shouldn't this be under unreleased?

Reply:
you are right, my bad i didn't pay attention!
i will correct it in the next one https://github.com/openvinotoolkit/anomalib/pull/2348

Reply:
No worries! I missed it in the review as well